setup.py

```diff
@@ -8,6 +8,8 @@
 import ah_bootstrap
 from setuptools import setup
 
+ah_bootstrap._check_submodule("asdf-standard")
+
```

Comment:
Ah, so this is the magic incantation! Is it documented somewhere (e.g. in the instructions for using the package-template)?